CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+# Overview
+
+This documents explains the processes and practices recommended for
+contributing enhancements to the Prometheus charm.
+
+- It is recommended that prior to developing enhancements to this charm
+  an issue explaining the use case is created at the official
+  Canonical [github repository](https://github.com/canonical/prometheus-operator).
+- Should you require any assistance in exploring use cases or
+  discussing proposed implementation you may directly engage with
+  other developers in the Canonical
+  [public channel](https://chat.charmhub.io/charmhub/channels/charm-dev) using
+  [Mattermost](https://mattermost.com/).
```

Comment:
"- If you would like to chat with us about your use-cases or proposed implementation, you can reach us at Canonical
  [Canonical's Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev)."

Reply:
@balbirthomas do we actually monitor this?

Reply:
I am in that channel and do check it every once in a while. If someone mentions my name the response is likely to be immediate. In case someone asks a question and I fail to notice I would hope someone will mention my name.